docs/spellchecker-dictionary

```diff
@@ -129,4 +129,12 @@ inferenceClientUrl
 annotationLevel
 AnnotationLevel
 progressbar
-macOS
\ No newline at end of file
+macOS
+SDK
+avd
+ADBKeyboard
+base64
+Accelerometer
+android-sdk-tools
+adb
+ADB
```

Comment:
I absolutely love that you added the dict so that we achieve consistency there :) 

Btw., empty line missing at eof

Reply:
Added